deepfence_agent/Dockerfile

```diff
@@ -52,41 +52,45 @@ COPY etc/fenced_logrotate.conf /etc/logrotate.d/fenced_logrotate.conf
 COPY etc/certs/* /etc/filebeat/
 COPY start_agent.sh /usr/local/bin/start_agent
 COPY tools/apache/scope/docker/deepfence_exe /usr/local/discovery/deepfence-discovery
-COPY tools/apache/compliance_check /usr/local/bin/compliance_check
 COPY plugins/compliance/scripts /usr/local/bin/compliance_check/scripts
 COPY plugins/compliance/config.json /usr/local/bin/compliance_check/config.json
-COPY plugins/bin/compliance /usr/local/bin/compliance_check/compliance
 COPY tools/apache/scope/docker/agent_check.sh /home/deepfence/
 COPY supervisord.conf /home/deepfence/supervisord-temp.conf
 COPY run_discovery.sh /home/deepfence/
 COPY create_cgroups.sh /home/deepfence/create-cgroups.sh
-COPY plugins/bin /home/deepfence/bin
+RUN mkdir -p /home/deepfence/bin
```

Comment:
Can you do all mkdir in one statement? Too many docker layers is also a problem. Old issue: https://github.com/deepfence/enterprise-roadmap/issues/788